src/TwigExtension.php

```diff
@@ -93,11 +93,13 @@ public function renderUri($name, $parameters = [], $relative = false)
      */
     public function renderAssetUrl($path, $packageName = null, $absolute = false, $version = null)
     {
+        $assetVersion = ($version) ? $version : $this->assetsVersion;
```

Comment:
I tried but it didn't work. From the 2 tests I get this:

testRenderAssetUrlUsesComposedAssetUrlAndVersionToGenerateUrl:
0 = XYZ | NULL

Should be XYZ

testRenderAssetUrlUsesProvidedVersionToGenerateUrl
Y[[ = XYZ | ABC

Should be  ABC